packages/utils/src/backoff.ts

```diff
@@ -0,0 +1,42 @@
+import { durations } from './time'
+
+export async function wait(milliseconds: number): Promise<void> {
+  return new Promise((resolve) => setTimeout(resolve, milliseconds))
+}
+
+export async function backoff(
```

Comment:
Why don't add a description what to expect from this function?

Reply:
added